loader/src/main/scala/com/snowplowanalytics/snowplow/storage/bigquery/loader/Loader.scala

```diff
@@ -35,6 +36,9 @@ object Loader {
 
   implicit val coderBadRow: Coder[BadRow] = Coder.kryo[BadRow]
 
+  private val MetricsNamespace = "snowplow"
+  val latencyDistribution = ScioMetrics.distribution(MetricsNamespace, "latency_in_sec")
```

Comment:
Don't we want to make `"latency_in_sec"` configurable? I assume other apps/reporting will need this value and will be deployed by Terraform as well

Reply:
In what sense 'configurable' @benjben ? Do you mean the unit of measurement and / or metric namespace?

Reply:
I meant to get `"latency_in_sec"` from the parameters when the job starts instead of having it hard coded. This is in case there is a consumer of this metric also deployed with TF that needs to know the name of the metric

Reply:
Ah, I get it. I had considered it, and I think it will maybe end up being part of a more general overhaul of the BQ Loader config file. 

We already have some job params that can only be passed in via CL but not as part of the `config`. I didn't want to add more, as it creates confusion.

(The reason we keep adding CL params instead of updating the `config` file is that each `config` update requires a new schema on Iglu Central.)